### ISSUE TITLE
Windows installation is broken on "Read Consul package checksum" step

### ISSUE THREAD

**Issue by imcitius** [ansible-collections/ansible-consul#196]
Hello

I tried to install consul under Windows server 2012 Core, and get this kind of error:

```
TASK [consul-cluster.role : Read Consul package checksum] ************************************************************************************************************************************
task path: /home/citius/ansible/roles/internal/consul-cluster.role/tasks/install_windows.yml:29
Using module file /usr/lib/python2.7/site-packages/ansible/modules/windows/win_shell.ps1
<10.29.11.157> ESTABLISH WINRM CONNECTION FOR USER: administrator on PORT 5986 TO 10.29.11.157
EXEC (via pipeline wrapper)
fatal: [10.29.11.157]: FAILED! => changed=true 
  cmd: findstr consul_1.2.2_windows_amd64.zip /tmp/consul/consul_1.2.2_SHA256SUMS
  delta: '0:00:00.499999'
  end: '2018-08-29 03:35:16.065991'
  msg: non-zero return code
  rc: 1
  start: '2018-08-29 03:35:15.565991'
  stderr: |-
    FINDSTR: Cannot open consul_1.2.2_SHA256SUMS
  stderr_lines:
  - 'FINDSTR: Cannot open consul_1.2.2_SHA256SUMS'
  stdout: ''
  stdout_lines: <omitted>
        to retry, use: --limit @/home/citius/ansible/projects/windows/all.retry
```

File was in place, and the same command executed manually via cmd or powershell, worked OK, when executed at the same directory, but does not work from any other path.

So I suggest to add `chdir` parameter to `win_shell` doing this step:
```
- name: Read Consul package checksum
  win_shell: "findstr {{ consul_pkg }} /tmp/consul/consul_{{ consul_version }}_SHA256SUMS"
  args:
    chdir: /tmp/consul
  register: consul_pkg_checksum
  tags: installation
```

This change fixes the issue.

Thanks.

**Comment by imcitius** [comment]
Opened PR #198 with this fix.

**Comment by brianshumate** [comment]
Thanks for the report and PR!

**Comment by brianshumate** [comment]
I've merged and included this in the just released v2.0.7 of the role. Thanks again! I'll go ahead and close this now.
